lib/src/importer/node/implementation.dart

```diff
@@ -104,23 +109,35 @@ class NodeImporter {
       if (value != null) return _handleImportResult(url, previous, value);
     }
 
+    if (parsed.scheme == '' || parsed.scheme == 'file') {
+      var result = _resolveLoadPath(p.fromUri(parsed), previous);
+      if (result != null) return result;
```

Comment:
Could this just `return _resolveLoadPath(p.fromUri(parsed), previous);` since the function would return `null` anyway if it goes passed this block?

Reply:
It definitely could!